pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -155,8 +155,11 @@ def render(self, task) -> Text:
                 self._current_task_id = task.id
             if self._trainer.training and task.id != self._current_task_id:
                 return self._tasks[task.id]
-            _text = ""
 
+            if self._trainer.progress_bar_callback.main_progress_bar_id != task.id:
+                return Text("")
+
+            _text = ""
```

Comment:
Side question: Why is this prefixed?

```suggestion
            text = ""
```

Reply:
copy-pasted from above. didn't changed. we can remove the prefix.

Reply:
Yeah let's do it